src/wasm/kotlin/java/arcs/Particle.kt

```diff
@@ -215,6 +215,11 @@ open class Singleton<T : Entity<T>>(
         this.entity = entity
         val encoded = entity.encodeEntity()
         RuntimeClient.singletonSet(particle, this, encoded)
+        if (!entity.isSet()) {
+            log("WARNING: ${
+                entity.getFieldsNotSet().joinToString(", ")
+            } fields on $entity are not set")
```

Comment:
Do we maybe want something stronger than a warning? e.g. throwing an exception?

Reply:
Eventually yes, we went with a warning to start to not break things.